amr-wind/overset/TiogaInterface.cpp

```diff
@@ -105,14 +105,17 @@ void TiogaInterface::register_solution()
     m_qnode = repo.create_scratch_field(
         pres.num_comp(), pres.num_grow()[0], pres.field_location());
 
+    vel.fillpatch(0.0);
+    pres.fillpatch(0.0);
+
```

Comment:
This fillpatch isn't necessary as we do fillpatch after solve. Did you notice some discrepancies that required this fillpatch call?

Reply:
I did notice some discrepancies. I'm not sure why... I tried to remove it but had to put it back in. We should double check this.